pubnub-core/pubnub-core-api/src/commonMain/kotlin/com/pubnub/api/models/consumer/objects/channel/PNChannelMetadata.kt

```diff
@@ -1,12 +1,29 @@
 package com.pubnub.api.models.consumer.objects.channel
 
+import com.pubnub.api.utils.PatchValue
+
 data class PNChannelMetadata(
     val id: String,
-    val name: String?,
-    val description: String?,
-    val custom: Map<String, Any?>?,
-    val updated: String?,
-    val eTag: String?,
-    val type: String?,
-    val status: String?,
-)
+    val name: PatchValue<String?>?,
+    val description: PatchValue<String?>?,
+    val custom: PatchValue<Map<String, Any?>?>?,
+    val updated: PatchValue<String>?,
+    val eTag: PatchValue<String>?,
+    val type: PatchValue<String?>?,
+    val status: PatchValue<String?>?,
+) {
+    /**
+     * Merge information from this `PNChannelMetadata` with new data from `update`, returning a new `PNChannelMetadata` instance.
```

Comment:
What about adding unit test for this. Does it make sense?

Reply:
yeah, I forgot to do it. Will add